test/support/pages/visualize_page.js

```diff
@@ -400,11 +434,9 @@ define(function (require) {
       return this.remote
       .setFindTimeout(defaultTimeout)
       .findByCssSelector('div.y-axis-div-wrapper > div > svg > g > g:last-of-type')
-      .then(function setYAxisLabel(y) {
-        return y.getVisibleText();
-      })
+      .getVisibleText()
       .then(function (yLabel) {
-        yAxisLabel = yLabel.replace(',', '');
+        yAxisLabel = yLabel.replace(',', '').replace('%','');
```

Comment:
nitpick, can we add a comment explaining why we are replacing these?  if you want to do these in one replace we could do something .like `replace(/(%|\,)/g, '')